src/bio2bel/sources/intact.py

```diff
@@ -0,0 +1,725 @@
+# -*- coding: utf-8 -*-
+
+"""Download and convert IntAct to BEL.
+
+Run with ``python -m bio2bel.sources.intact``
+
+`IntAct <https://www.ebi.ac.uk/intact/>`_ is a interaction database with information about interacting proteins,
+their relation, and the experiments, in which these interactions were found.
+Among the interactions that are documented in IntAct are protein modifications, associations, direct interactions,
+binding interactions and cleavage reactions.
+These interactions were grouped according to their biological interpretation and mapped to the corresponding BEL
+relation. The interactions in IntAct had a higher granularity than the interactions in BioGRID. Due to the default BEL
+namespace of protein modifications :data:`pybel.language.pmod_namespace`, the post-translational protein modification
+can be identified very accurately. For example, the glycosylation of a protein can be described in BEL by
+:code:`pybel.dsl.ProteinModification('Glyco')`.  Although many protein modifications had corresponding terms in BEL,
+there were some interaction types in IntAct that could not be mapped directly, like 'gtpase reaction' or
+'aminoacylation reaction'.
+Therefore, other vocabularies like the `Gene Ontology (GO) <https://www.ebi.ac.uk/QuickGO/>`_ or the
+`Molecular Process Ontology (MOP) <https://www.ebi.ac.uk/ols/ontologies/mop>`_ were used to find corresponding
+interaction terms. These terms were then annotated with the name, namespace and identifier.
+For negative protein modifications in which a group is split from the protein like `decarboxylation reaction
```

Comment:
once you build the HTML locally and look at it, you will also notice that this is a wall of text. please break into appropriate paragraphs.